src/client/pydaos/pydaos_shim.c

```diff
@@ -1375,31 +1384,29 @@ __shim_handle__kv_iter(PyObject *self, PyObject *args)
 }
 
 static PyMethodDef daosMethods[] = {
-	/** Generic methods */
-	EXPORT_PYTHON_METHOD(daos_init),
-	EXPORT_PYTHON_METHOD(daos_fini),
-	EXPORT_PYTHON_METHOD(err_to_str),
-
-	/** Container operations */
-	EXPORT_PYTHON_METHOD(cont_open),
-	EXPORT_PYTHON_METHOD(cont_open_by_path),
-	EXPORT_PYTHON_METHOD(cont_get),
-	EXPORT_PYTHON_METHOD(cont_newobj),
-	EXPORT_PYTHON_METHOD(cont_close),
-	EXPORT_PYTHON_METHOD(cont_check),
-	EXPORT_PYTHON_METHOD(cont_check_by_path),
-
-	/** KV operations */
-	EXPORT_PYTHON_METHOD(kv_open),
-	EXPORT_PYTHON_METHOD(kv_close),
-	EXPORT_PYTHON_METHOD(kv_get),
-	EXPORT_PYTHON_METHOD(kv_put),
-	EXPORT_PYTHON_METHOD(kv_iter),
-
-	/** Array operations */
-
-	{NULL, NULL}
-};
+    /** Generic methods */
+    EXPORT_PYTHON_METHOD(daos_init),
+    EXPORT_PYTHON_METHOD(daos_fini),
+
+    /** Container operations */
+    EXPORT_PYTHON_METHOD(cont_open),
+    EXPORT_PYTHON_METHOD(cont_open_by_path),
+    EXPORT_PYTHON_METHOD(cont_get),
+    EXPORT_PYTHON_METHOD(cont_newobj),
+    EXPORT_PYTHON_METHOD(cont_close),
+    EXPORT_PYTHON_METHOD(cont_check),
+    EXPORT_PYTHON_METHOD(cont_check_by_path),
+
+    /** KV operations */
+    EXPORT_PYTHON_METHOD(kv_open),
+    EXPORT_PYTHON_METHOD(kv_close),
+    EXPORT_PYTHON_METHOD(kv_get),
+    EXPORT_PYTHON_METHOD(kv_put),
+    EXPORT_PYTHON_METHOD(kv_iter),
+
+    /** Array operations */
+
+    {NULL, NULL}};
```

Comment:
Tabs changed to spaces?

Reply:
Possibly, `err_to_str` is removed and as a result the block got re-indented.

Reply:
Shouldn't it be tabs?

Reply:
Not unless the formatting tool is broken.

Reply:
Our convention is tabs so that means the tool is broken?

Reply:
This is a struct definition rather than code don't forget.

Reply:
Why does struct definition uses spaces but otherwise code indentation uses tabs? That's seems really confusing